CITATION.cff

```diff
@@ -0,0 +1,18 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Resende"
+  given-names: "Ximenes"
+- family-names: "de Sá"
```

Comment:
after the first author, the others must be in alphabetical order.
I think you missed Afonso Mukai and Luana Vilela as authors too.

Reply:
Based on the discussion on https://github.com/lnls-fac/pyaccel/pull/109, we decided to attempt following the order of contribution in the authors.

I adjusted the author's list following the order of pyaccel.

Thanks!